cg/apps/vogue.py

```diff
@@ -0,0 +1,60 @@
+"""
+    Module for vogue API
+"""
+
+import json
+import logging
+import subprocess
+from subprocess import CalledProcessError
+
+LOG = logging.getLogger(__name__)
+
+
+class VogueAPI():
+
+    """
+        API for vogue
+    """
+
+    def __init__(self, config: dict):
+        super(VogueAPI, self).__init__()
+        self.vogue_binary = config['vogue']['binary_path']
+        self.base_call = [self.vogue_binary]
+
+    def load_genotype(self, genotype_dict: dict):
```

Comment:
I suggest renaming this method to add_observations_from(self, vcf: dict) according to the docstring, this to avoid hard and soft couplings between our different applications, what do you say?